lib/std/debug.zig

```diff
@@ -1097,15 +1097,29 @@ fn readCoffDebugInfo(allocator: mem.Allocator, coff_obj: *coff.Coff) !ModuleDebu
         const len = try coff_obj.getPdbPath(path_buf[0..]) orelse return di;
         const raw_path = path_buf[0..len];
 
-        const path = try fs.path.resolve(allocator, &[_][]const u8{raw_path});
-        defer allocator.free(path);
+        di.pdb = blk: {
+            if (fs.path.isAbsolute(raw_path)) {
+                if (pdb.Pdb.init(allocator, raw_path)) |p| {
+                    break :blk p;
+                } else |err| switch (err) {
+                    error.FileNotFound, error.IsDir => {},
+                    else => return err,
+                }
+            }
 
-        di.pdb = pdb.Pdb.init(allocator, path) catch |err| switch (err) {
-            error.FileNotFound, error.IsDir => {
-                if (di.dwarf == null) return error.MissingDebugInfo;
-                return di;
-            },
-            else => return err,
+            var self_dir_buf: [windows.MAX_PATH]u8 = undefined;
+            const self_dir = try fs.selfExeDirPath(&self_dir_buf);
```

Comment:
This looks like it will always get a completely wrong path. Why do you think this will work?

Reply:
When distributed together, the pdb is usually stored in the same directory as the exe.

Reply:
I apologize. I think when I reviewed this yesterday, I was confused and thought this file path corresponded to a source file, not the .pdb file.